ethereum/statetransition/src/test/java/tech/pegasys/artemis/statetransition/util/BeaconStateUtilTest.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.statetransition.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.google.common.primitives.UnsignedLong;
+import net.consensys.cava.junit.BouncyCastleExtension;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(BouncyCastleExtension.class)
+class BeaconStateUtilTest {
```

Comment:
How about some cases for min of same value, max of same value?

Reply:
done